script/.contributors_aliases.json

```diff
@@ -0,0 +1,153 @@
+{
+  "Areveny": {
+    "authoritative_mail": "areveny@protonmail.com",
+    "mails": ["areveny@protonmail.com", "self@areveny.com"],
+    "team": "true"
```

Comment:
A true / false value here is quite inflexible. What about the actual team / group name? That would allow for more than two groups if we ever want that.

Reply:
[c8f4f3b](https://github.com/PyCQA/astroid/pull/1441/commits/c8f4f3b5ab2004b03a0dd2a23c1626b791cc8029)